M2/Macaulay2/d/version.dd

```diff
@@ -162,7 +163,7 @@ setupconst("version", Expr(toHashTable(Sequence(
 	       \"not present\"
 	  #endif
 	  "),
-   "mpsolve version" => "3.1.8", -- TODO: DETERMINE VERSION #??
+   "mpsolve version" => Ccode(constcharstar, "stringize(MPS_MAJOR_VERSION) \".\" stringize(MPS_MINOR_VERSION) \".\" stringize(MPS_PATCH_VERSION)" ),
```

Comment:
Could you add something like this?:
```
#ifndef MPS_MAJOR_VERSION
#error "MPS_MAJOR_VERSION undefined"
#endif
```
That will make sure we are getting the actual numbers to display.

Reply:
If I add that now it'll lead to errors in the autotools build, since currently it builds an older version of MPSolve that doesn't have those macros. Would you like to add the check along with updating MPSolve?

Reply:
I added a different fix for now.